app/src/components/main/modal/modalLayout/modalContent/modalContent.jsx

```diff
@@ -5,14 +5,16 @@ import styles from './modalContent.scss';
 
 const cx = classNames.bind(styles);
 
-export const ModalContent = ({ children }) => (
-  <div className={cx('modal-content')}>
+export const ModalContent = ({ children, permissionMap }) => (
```

Comment:
General component `ModalContent` should know nothing about permissions and other content.
If you need to override styles for a specific case, I suggest you to provide `className` in the props and use it along with the current class name:
`className={cx('modal-content', className)}`